test/1_stdlib/Float.swift

```diff
@@ -222,13 +222,24 @@ func checkQNaN(_ qnan: TestFloat) {
   _precondition(qnan.floatingPointClass == .quietNaN)
 }
 
+func checkSNaN(_ snan: TestFloat) {
+  checkNaN(snan)
+// FIXME: i386 doesn't fully support signaling NaN.
```

Comment:
`FIXME` is a bit misleading here; i386 *cannot* fully support signaling NaN without a massive change to the calling conventions (because loading a Float or Double sNaN to x87 always signals invalid and quiets the NaN).  The comment should read something more like:

`//  `sNaN` cannot be fully supported on i386.`